standalone_tests/artifact_object_reference_test.py

```diff
@@ -188,10 +188,22 @@ def _make_wandb_table():
 def _make_wandb_joinedtable():
     return wandb.JoinedTable(_make_wandb_table(), _make_wandb_table(), "id")
 
+def _make_wandb_audio():
```

Comment:
Do you mind adding the Audio to the `_make_wandb_table` function (so basically add an entry to the `columns` array called "Audio", then add this `_make_wandb_audio()` entries to the data of `_make_wandb_table`?

Reply:
yep; done